servicebroker/config/settings.json

```diff
@@ -1,5 +1,7 @@
 {
   "port": 8080,
   "user": "user",
```

Comment:
ok, will do the rebase, and also move the createService part out of the PR.   For create Service part,  I will raise another PR, and change the style for Sequelize call